CONTRIBUTING.md

```diff
@@ -104,6 +104,7 @@ import common
 from utils import rootfs
 ```
 - Minimize [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity). Most python style checkers also come with McCabe complexity checkers. A good rule of thumb is to limit the number of if-then-else statements to 3 and return once in a module.
+- Tern uses `prospector` to analyze the source code for style issues.  Contributions should resolve any new issues reported.  `prospector` is available via the `dev-requirements.txt` pip package file, i.e., `pip install -r dev-requirements.txt`.
```

Comment:
I was actually thinking of adding a whole section on setting up your development environment, and including the installation of prospector in there for checking for coding errors. You can add that section if you want. Here's how I do it:
1. Run `python3 -m venv terndevel`
2. `cd` into the directory and run `source bin/activate`
3. git clone your fork
4. Run `pip install -r dev-requirements.txt` to get prospector installed
5. Make changes
6. Run `prospector .`
7. Fix any style issues
8. Commit changes and submit PR